gapic/src/main/com/google/gapid/perfetto/models/Selection.java

```diff
@@ -116,6 +116,10 @@ public Composite buildUi(Composite parent, State state) {
           (Selection<Key>) selections.get(type) : Selection.emptySelection();
     }
 
+    public NavigableMap<Kind<?>, Selection<?>> getSelections() {
+      return selections;
+    }
+
```

Comment:
Instead of exposing the map, add an `addSelection(MultiSelection)` method here.

Reply:
Done.